src/v2/Apps/Fair/Components/FairExhibitorSortFilter.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import { SortFilter } from "v2/Components/SortFilter"
+import { useExhibitorsFilterContext } from "../Routes/ExhibitorFilterContext"
```

Comment:
**suggestion**: What do you think of moving this context component to `Fair/Components` instead of `Fair/Routes`? I think routes are usually pretty specific to the routing layer.

Reply:
That's a good point, I'll move it. Thanks!